pkg/nfd-master/nfd-master.go

```diff
@@ -50,6 +50,11 @@ import (
 	"sigs.k8s.io/node-feature-discovery/pkg/version"
 )
 
+const (
+	defaultDenyNs    = "kubernetes.io"
```

Comment:
nit: If we specify consts it would be probably better name these so that these it's clear that they're meant for annotations

```suggestion
	defaultDenyAnnotationNs    = "kubernetes.io"
```

Reply:
Maybe we could have a generic default, I am thinking also in ExtendedResources, I need the same NS to deny, so having it as a generic default makes some sense

Reply:
So I reserve the name of `defaultDenyNs `